content/en/integrations/guide/source-code-integration.md

```diff
@@ -28,72 +24,60 @@ With the source code and GitHub Apps integrations, you can also see inline code
 
 ## Configuration
 
-To map telemetry data with your source code, you need to send metadata to Datadog from your CI pipeline.
-
-To tag your process with the commit SHA, you need a container label, Kubernetes label, or annotation.
+<div class="alert alert-info">
+The source code integration supports Go and all JVM languages.
+<br>
+You need to be using Datadog Agent 7.33.0 or higher.
+</div>
 
-1. Set a container tag or extend the `DD_TAGS` environment variable.
-2. Upload your git metadata including the commit SHAs and your git repository URL by running [`datadog-ci git-metadata upload`][1].
-3. Optionally, [install GitHub Apps][2].
+To map telemetry data with your source code, you need to:
 
-Datadog correlates the places where you can link directly to your git repository.
+1. Add `git.commit.sha` and `git.repository_url` tags to your containers, or directly on your telemetry.
+2. Upload metadata about your git repository by running [`datadog-ci git-metadata upload`][1] in your CI pipeline.
+3. Optionally, [install a GitHub App][2] to display inline source code snippets.
 
-### Tag your builds
+### Tag your telemetry
 
-To get direct links from your stacktrace to your git repository, tag your telemetry with a `git.commit.sha` tag.
+To link data to a specific commit, tag your telemetry with a `git.commit.sha` tag.
 
 {{< tabs >}}
-{{% tab "Docker Runtime" %}}
-
-If your containers are running on Docker, Datadog can extract the commit SHA directly from your images' Docker labels.
+{{% tab "Containers" %}}
 
+<div class="alert alert-warning">
+This approach requires Docker, or containerd >= 1.5.6. For other container setups, see "Other".
```

Comment:
...where is "Other"? there was one in the K8s tab but it looks like it's been deleted/replaced.

Reply:
It's referencing the third "Other" tab: https://docs-staging.datadoghq.com/ronan/sci-docs-improvements/integrations/guide/source-code-integration/?tab=other#tag-your-telemetry

Maybe there is a better way to do it, by having a link that switches the tab directly ?

Reply:
Ah ok I just missed the beginning of that tab when I was looking through the source. I'm not sure if there's a way to link to a tab from a tab, but I think we could probably just add some more context here at the end of the sentence
```suggestion
This approach requires Docker, or containerd >= 1.5.6. For other container setups, see the "Other" section.
```